package/tailscale-cert-renewal.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Tailscale Certificate Renewal
+After=network.target tailscaled.service
+
+[Service]
+Type=oneshot
+ExecStart=/bin/sh -c 'for cert in /data/tailscale/certs/*.crt; do [ -f "$cert" ] && hostname="${cert##*/}" && hostname="${hostname%.crt}" && /data/tailscale/manage.sh cert renew "$hostname"; done'
```

Comment:
Is this necessary? There should only be one valid certificate since only one hostname is valid - if we derive the hostname automatically using `tailscale status --peers=false --json | jq -r '.Self.DNSName[:-1]'` within the `manage.sh cert` subcommand then we can just run `manage.sh cert renew` here without needing to specify arguments.

Reply:
You're right, simplified to just `/data/tailscale/manage.sh cert renew` 👍